packages/block-editor/src/hooks/block-hooks.js

```diff
@@ -151,6 +171,18 @@ function BlockHooksControlPure( { name, clientId } ) {
 					false
 				);
 				break;
+
+			default:
+				// If we do not know the relative position, it is because the block was
+				// added via a filter. In this case, we default to inserting it after the
+				// current block.
+				insertBlock(
+					block,
+					blockIndex + 1,
+					rootClientId, // Insert as a child of the current block's parent
+					false
+				);
+				break;
```

Comment:
What does "not knowing" the relative position mean? Is it omitted? Do we have `null` or `undefined` here?

I ask because that's different from having an invalid relative position, and this change starts to handle invalid positions as well as the "unknown" position.

For example, it looks like a block with `realtivePosition: 'alpha_centauri'` would not have been inserted before, but it would with this change.

Reply:
Ah, that's a good point. It is omitted (unavailable really), so it's probably `undefined` (or `null`, I'll have to check this).

Good point to rule out neighboring stars 😬, so I might change the `default` to a `case undefined:` 👍

Reply:
https://github.com/WordPress/gutenberg/pull/59396/commits/addb757c9fdf444a61c2de5993541db8af85f88c